### PR TITLE
feat(scss) component page-header

### LINKED ISSUE #1308: Update Header
Update header to new Styleguides

### PR DESCRIPTION
**Pull request by almonais**
fixes #1308 

image

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1392

**Review by sanlucca — DISMISSED** [review]
The breadcrumbs should not take space. They live in the whitespace between the banner and the title (there is always 40px between the banner and the title)

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1392

File: packages/scss/src/theming/_commons.scss [hunk 1]
```diff
@@ -68,7 +68,8 @@ $commons: (
 		background: _color("primary", "200"),
 		frontground: _color("primary", "color"),
 		speed: 600ms,
-	)
+	),
+	padding: 1.5rem 2.5rem 0 2.5rem,
```
Comment (sanlucca) [inline packages/scss/src/theming/_commons.scss:72]: What is that padding used for ? Doesn't looks like a "common" var

File: packages/scss/src/components/_page-header.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	padding-bottom: _theme('spacings.standard');
+	background-color: _color('white');
+
+	.menu {
+		&::after {
+			display: none;
+		}
+	}
+
+	&.mod-withBreadcrumbs {
```
Comment (AlexiaAlarza) [inline packages/scss/src/components/_page-header.scss:13]: Can we suppose that all breadcrumbs inside a .page-header will have a margin-bottom 0 ? Like that we can delete the "mod-withBreadcrumbs"

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1392

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1392

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1392

File: packages/scss/src/components/_page-header.scss [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+.pageHeader-content-actions {
+	& > *:not(:last-child){
+		margin-right: _theme('spacings.small');
+	}
+}
+
+.pageHeader-description {
+	max-width: 50rem;
```
Comment (BertrandPodevin) [inline packages/scss/src/components/_page-header.scss:46]: Why 50rem ? This is a strange arbitrary value. Maybe we should link it to some kind of responsive, using % unit or make it themable ?

File: packages/scss/src/theming/_commons.scss [hunk 4]
```diff
@@ -68,7 +68,7 @@ $commons: (
 		background: _color("primary", "200"),
 		frontground: _color("primary", "color"),
 		speed: 600ms,
-	)
+	),
```
Comment (BertrandPodevin) [inline packages/scss/src/theming/_commons.scss:71]: Remove this comma

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1392

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1392

File: packages/scss/src/components/_page-header.scss (hunk 3 above)
Comment (AlexiaAlarza) [reply packages/scss/src/components/_page-header.scss:42]: this is about the recommendation on design that we must display less than 40 words i think on the same line ?

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1392

File: packages/scss/src/components/_page-header.scss [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+.pageHeader-content-actions {
+	& > *:not(:last-child){
+		margin-right: _theme('spacings.small');
+	}
+}
+
+.pageHeader-description {
+	max-width: 50%;
```
Comment (BertrandPodevin) [inline packages/scss/src/components/_page-header.scss:42]: A flat 50% will be problematic in a responsive context. We should keep the max-width: 50rem but make it themable

**Review by AlexiaAlarza — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1392

**Pull request merged**